new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,42 @@
+import downloadDecompressedLogs from "../utils/downloadDecompressedLogs";
+
+
+class LogExportManager {
+    private blob: Blob;
+
+    private numChunks: number;
+
+    constructor (numChunks?: number) {
+        this.blob = new Blob();
+        this.numChunks = numChunks ?? 0;
+    }
+
+    getBlob (): Blob {
+        return this.blob;
+    }
```

Comment:
Do we have to expose the blob here? Can we do a check in `appendChunkData` to see if we have received enough chunks and trigger the download if that's the case?